src/CircleGraph/CircleViewer.ts

```diff
@@ -83,10 +91,27 @@ export class CircleViewerDocument implements vscode.CustomDocument {
           this._circleViewer.splice(index, 1);
         }
       });
+      if (this._fileWatcher) {
+        this._fileWatcher.dispose();
+        this._fileWatcher = undefined;
+      }
     });
 
     return view;
   }
+
+  public reload(panel: vscode.WebviewPanel) {
+    if (this._reloadTimer) {
+      clearTimeout(this._reloadTimer);
+    }
+    this._reloadTimer = setTimeout(() => {
+      this._circleViewer.forEach((view) => {
+        if (view.owner(panel)) {
+          view.loadContent();
+        }
+      });
+    }, 500);
```

Comment:
delay with timer is used to not to make too many reloads while model is in some continuous update.

Reply:
(optional) Could you add that as a comment..? It will be helpful to understand the code later.